### PR TITLE
Minor updates to README.md files

### PR DESCRIPTION
**Pull request by james-laing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,4 +170,4 @@ Drill down into specific arrays and identify top busy hosts while correlating re
 <img src="extra/grafana/images/grafana_purefa_overview_dash_1.png" width="66%" height="66%">
 <img src="extra/grafana/images/grafana_purefa_overview_dash_2.png" width="33%" height="33%">
 <br>
-For more information on dependencies and notes to deploy, take look at the files in the [extra/grafana/](extra/grafana/) folder and [README.md](extra/grafana/README.md)
+For more information on dependencies and notes to deploy, take look at the files in the [extra/grafana/](extra/grafana/) folder and [README.md](extra/grafana/README.md) and an example of prometheus.yaml configuration can be found in [extra/prometheus/](extra/prometheus/) folder.
```
Comment (chrroberts-pure) [inline README.md:173]: This sentence kind of runs on, I would re-word closer to:

`For more information on dependencies, and notes to deploy -- take look at the examples for Grafana and Prometheus in the [extra/grafana/](extra/grafana/) and [extra/prometheus/](extra/prometheus/) folders respectively.`

Happy to be overridden and @sdodsley is good about wordsmithing too

File: extra/grafana/README.md [hunk 2]
```diff
@@ -6,10 +6,9 @@ This exporter is provided under Best Efforts support by the Pure Portfolio Solut
 
 ## TL;DR
 1. Configure Pure Storage OpenMetrics Exporter ([pure-fa-openmetrics-exporter][1]).
-2. Deploy and configure Prometheus ([prometheus-docs][2]).
+2. Deploy and configure Prometheus ([prometheus-docs][2]). Example [prometheus.yaml](../prometheus/prometheus.yaml) here.
```
Comment (chrroberts-pure) [inline extra/grafana/README.md:9]: `[Example prometheus.yaml here](../prometheus/prometheus.yaml)`

File: README.md [hunk 3]
```diff
@@ -159,7 +159,7 @@ A complete example monitoring stack implemented in Docker Compose which can be f
 |purefa_volume_space_data_reduction_ratio    |FlashArray volume space data reduction               |
 
 ## Monitoring On-Premise with Prometheus and Grafana
-Take a holistic overview of your Pure Storage FlashArray estate on-premise with Prometheus and Grafana to summerise statistics such as:
+Take a holistic overview of your Pure Storage FlashArray estate on-premise with Prometheus and Grafana to summarize statistics such as:
```
[the remark this answers is not in the document]
Comment (sdodsley) [reply README.md:162]: +1

File: README.md (hunk 1 above)
Comment (sdodsley) [reply README.md:173]: Agree with @chrroberts-pure review

File: extra/grafana/README.md (hunk 2 above)
Comment (sdodsley) [reply extra/grafana/README.md:9]: +1

File: extra/grafana/README.md [hunk 4]
```diff
@@ -74,7 +73,7 @@ svc-readonly  local  a12345bc6-d78e-901f-23a4-56b07b89012  2022-11-30 08:58:40 E
 ```
 
 3. Configure `/etc/prometheus/prometheus.yaml` to point use the OpenMetrics exporter to query the device endpoint.
-An example of a single FlashArray device configuration is here: [](../prometheus/prometheus.yaml)
+An example of configuring [prometheus.yaml](../prometheus/prometheus.yaml) for scraping FlashArray endpoints is here.
```
Comment (sdodsley) [inline extra/grafana/README.md:76]: [This is an example of configuring the prometheus.yaml.](../prometheus/prometheus.yaml)

File: extra/grafana/README.md [hunk 5]
```diff
@@ -156,13 +155,11 @@ Check the data is accessible to each component in the stack. If at any on these
...
-  - Authorization > Type:Bearer Token: `a12345bc6-d78e-901f-23a4-56b07b89012`.
-<br>
-<img src="./images/postman-purefa-openmetricsexporter-query.png" width="40%" height="40%">
-<br>
-2. Ensure the API authorization token is correct and correctly configured in prometheus.yaml.
+1. Run cURL against the exporter and pass is the bearer token and endpoint. 
+```
+curl -H 'Authorization: Bearer a12345bc6-d78e-901f-23a4-56b07b89012' -X GET 'http://<exporter_ip>:9490/metrics/array?endpoint=arrayname01.fqdn.com'
+```
+
```
[the remark this answers is not in the document]
Comment (sdodsley) [reply extra/grafana/README.md:162]: All following bullet point number need to decrement by 1

**Review by sdodsley — CHANGES_REQUESTED** [review]

File: extra/grafana/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (james-laing) [reply extra/grafana/README.md:163]: Done

File: README.md (hunk 1 above)
Comment (james-laing) [reply README.md]: Yes, this makes sense, thanks folks.

File: extra/grafana/README.md (hunk 4 above)
Comment (james-laing) [reply extra/grafana/README.md]: Done - thanks

**Comment by james-laing** [comment]
Above changes have been implemented in the latest commit. Thanks folks.

**Comment by chrroberts-pure** [comment]
LGTM

**Review by sdodsley — APPROVED** [review]
